tests/rust/tcp.rs

```diff
@@ -459,7 +459,7 @@ fn tcp_bad_listen() {
         Err(e) if e.errno == libc::EINVAL => (),
         _ => panic!("invalid call to listen() should fail with EINVAL"),
     };
-    safe_close_passive(&mut libos, sockqd);
+    safe_close_active(&mut libos, sockqd);
```

Comment:
Inside the safe_close_active function, the error still says passive socket.

Reply:
Fixed.